nltk/tokenize/casual.py

```diff
@@ -67,6 +67,7 @@
       [<>]?
       |
       <3                         # heart
+      </3                        # Broken-heart
```

Comment:
This does not work. It will try to match `<3</3`. Merging this would prevent the TweetTokenizer from recognising hearts.

What could work is 
```python
...
|
<3                   # heart
|
</3                  # broken heart
```
But at that point, just use regex:
```python
...
|
</?3                 # heart, perhaps broken
```

Reply:
Yes, I totally forgot that, thanks!